cocos/renderer/core/gfx/GFXCommandBuffer.h

```diff
@@ -13,7 +13,7 @@ class CC_CORE_API GFXCommandBuffer : public GFXObject {
 public:
   virtual bool initialize(const GFXCommandBufferInfo& info) = 0;
   virtual void destroy() = 0;
-  virtual void begin() = 0;
+  virtual void begin(GFXRenderPass* renderPass = nullptr, uint subpass = 0, GFXFramebuffer* frameBuffer = nullptr) = 0;
```

Comment:
need modify MTLCommandBuffer::begin as well.

Reply:
it seems that all changes does not apply to metal

Reply:
I don't have the environment to check it so... I figured it would be better if someone else fill in those gaps.

Reply:
please let me know if you need me to fix the metal fix, or otherwise CI will failed

Reply:
I've pushed all the changes I can think of, but I'm working in the dark here, so please verify the changes and if needed, submit a fix PR to this branch.